app/src/test/kotlin/com/getstrm/pace/processing_platforms/postgres/PostgresViewGeneratorTest.kt

```diff
@@ -379,33 +376,33 @@ with
     )
   )
 select
-  transactionid,
+  "transactionid",
   case
-    when ('fraud_and_risk' IN ( SELECT rolname FROM user_groups )) then userid
+    when ('fraud_and_risk' IN ( SELECT rolname FROM user_groups )) then "userid"
     else 123
   end as userid,
   case
     when ('marketing' IN ( SELECT rolname FROM user_groups )) then regexp_replace(email, '^.*(@.*)${'$'}', '****\1', 'g')
-    when ('fraud_and_risk' IN ( SELECT rolname FROM user_groups )) then email
+    when ('fraud_and_risk' IN ( SELECT rolname FROM user_groups )) then "email"
```

Comment:
why no quotes around user_groups and rolname ?

Reply:
I'll update that as well. Didn't want to go through all SQL DDL again 🙃